recipe/build.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
```

Comment:
One of things we have tried setting in the past is `DYLD_FALLBACK_LIBRARY_PATH`. Though `sqlite` seems to be pretty stubborn. So maybe `DYLD_LIBRARY_PATH` would be better. The difference between the two is a matter of priority. There seems to be a `READLINE_LIBS` environment variable for `sqlite`. Would give it `readline` and `ncurses`. Unfortunately `readline` doesn't support `pkg-config` so this will all need to be manually done.